cp/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/OurTaskService.java

```diff
@@ -22,6 +23,17 @@
 
   public List<TaskDto> readAll() {
     Bundle tasksBundle = taskRepository.findAllOurTasks();
+    FhirUtil.getFromBundle(tasksBundle, Task.class)
+        .forEach(task -> {
+          Task baseTask = taskRepository.find(task.getBasedOn()
```

Comment:
us _include/_revinclude instead of separate request per resource. this is inefficient.